### PR TITLE
Update UG and DG

### PR DESCRIPTION
**Pull request by RadieonAjax**

### CODE REVIEW

File: docs/diagrams/CommitActivityDiagram.puml [hunk 1]
```diff
@@ -3,16 +3,28 @@ skin rose
...
+    else ([Other Commands])
+      :Purge redundant slide;
+      :Execute the command;
+      :Save current state to stateList;
+    endif
+  endif
 endif
-stop
+
+end
```
Comment (tanyqcollin) [inline docs/diagrams/CommitActivityDiagram.puml:29]: try to change it to stop instead of end

**Review by yongqqqq — APPROVED** [review]
LGTM!

File: docs/diagrams/CommitActivityDiagram.puml (hunk 1 above)
Comment (RadieonAjax) [reply docs/diagrams/CommitActivityDiagram.puml]: Thanks for the heads up, it has been rectified!

**Review by Jianwen0451 — APPROVED** [review]
LGTM!

**Pull request merged**